types/distributions/index.d.ts

```diff
@@ -0,0 +1,22 @@
+// Type definitions for distributions 2.0
+// Project: https://github.com/AndreasMadsen/distributions
+// Definitions by: Marco Lanaro <https://github.com/marcolanaro>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+
+export interface Distribution {
+    pdf: (x: number) => number;
+    cdf: (x: number) => number;
+    inv: (p: number) => number;
+    mean: () => number;
+    median: () => number;
+    variance: () => number;
+}
+
+declare const distributions: {
+    Normal: (mean?: number, sd?: number) => Distribution;
+    Studentt: (df: number) => Distribution;
+    Uniform: (a?: number, b?: number) => Distribution;
+    Binomial: (properbility: number, size: number) => Distribution;
+};
+
+export default distributions;
```

Comment:
distributions doesn't have a default export -- it's a commonjs export. I think you should just export Normal, Studentt, etc individually:

```ts
export function Normal(mean?: number, sd?: number): Distribution;
// etc
```

And then you can go back to `import * as distributions from 'distributions';`